service/gcs/core/gcs/storage.go

```diff
@@ -177,6 +181,51 @@ func (c *gcsCore) unmountMappedVirtualDisks(disks []prot.MappedVirtualDisk) erro
 	return nil
 }
 
+// mountMappedDirectories mounts the given mapped directories using a Plan9
+// filesystem with the given options.
+func (c *gcsCore) mountMappedDirectories(dirs []prot.MappedDirectory) error {
+	for _, dir := range dirs {
```

Comment:
It's probably a good idea to have cleanup logic there just in case. Same for mountMappedVirtualDisks as well. I didn't include that in this change because adding proper cleanup logic across the GCS seemed like a PR all to itself. Maybe add an issue for that, and we'll tackle it soon?